src/docm/index.js

```diff
@@ -178,6 +178,12 @@ const processRecord = async (opt) => {
     if (!variant) {
         throw new Error('Failed to parse either variant');
     }
+    // get the vocabulary term
+    const relevance = await conn.getVocabularyTerm('recurrent');
+
```

Comment:
> try/catch at line 188 by replacing the diseaseRec.tags[0] by 'recurrent'
That is true to minimalize changes, however, wouldn't we have to call `getVocabularyTerm` as we loop through this loop? Figured we can just call the relevance object once

> I would suggest to add a reference to the KBDEV-1050 ticket in a comment since it will seems unusual to use a fix relevance term, maybe on line 189.

Yessir

> Last thing, this is more like a bugfix than a new feature since we are fixing existant malfunctionning feature, so the branch could have been named accordingly.

Will do